cartography/data/jobs/cleanup/aws_dns_cleanup.json

```diff
@@ -11,9 +11,28 @@
       "iterationsize": 100
     },
     {
-      "query": "MATCH ()<-[r:DNS_POINTS_TO]-(:AWSDNSRecord)-[:MEMBER_OF_DNS_ZONE]->(:AWSDNSZone)<-[:RESOURCE]-(:AWSAccount{id: {AWS_ID}}) WHERE r.lastupdated <> {UPDATE_TAG} WITH r LIMIT {LIMIT_SIZE} DELETE (r) return COUNT(*) as TotalCompleted",
+      "query": "MATCH (:AWSDNSRecord)<-[r:DNS_POINTS_TO]-(:AWSDNSRecord)-[:MEMBER_OF_DNS_ZONE]->(:AWSDNSZone)<-[:RESOURCE]-(:AWSAccount{id: {AWS_ID}}) WHERE r.lastupdated <> {UPDATE_TAG} WITH r LIMIT {LIMIT_SIZE} DELETE (r) return COUNT(*) as TotalCompleted",
```

Comment:
This cleanup job runs at the [end of the route53 module](https://github.com/lyft/cartography/blob/c39a3bbfc1e8a7f8a6a04658467508b644db1c89/cartography/intel/aws/route53.py#L338) for a single AWS account.  The route53 sync creates the following `DNS_POINTS_TO` relationships:

- `(:AWSDNSRecord)-[:DNS_POINTS_TO]->(:AWSDNSRecord)`
- `(:AWSDNSRecord)-[:DNS_POINTS_TO]->(:LoadBalancer)`
- `(:AWSDNSRecord)-[:DNS_POINTS_TO]->(:EC2Instance)`
- `(:AWSDNSRecord)-[:DNS_POINTS_TO]->(:NameServer)`

As such, this cleanup job now explicitly deletes nodes with these labels instead of deleting all node types connected to AWSDNSRecords.